src/continuous-validation/body-pair-collision.cc

```diff
@@ -206,10 +207,19 @@ bool BodyPairCollision::computeDistanceLowerBound(
       return false;
     }
     if (result.distance_lower_bound < distanceLowerBound) {
+      iSmallest = i;
       distanceLowerBound = result.distance_lower_bound;
       assert(distanceLowerBound > 0);
     }
   }
+  if (distanceLowerBound < 0.001) {
```

Comment:
The lower bound threshold is this 0.001 in this draft PR. It would be made modifiable if the principle is accepted.